orion-server/src/main/java/com/pinterest/orion/core/automation/sensor/kafka/KafkaTopicSensor.java

```diff
@@ -154,30 +154,24 @@ public void populateTopicMetrics(KafkaCluster cluster, Map<String, KafkaTopicDes
       // Topic size
       double topicSize = getTopicSizeByteFromTopicDescription(topicDescription);
       OrionServer.metricsGaugeNum(
-              "kafkaTopic",
-              "size",
-              "byte",
-              metricsTags,
-              topicSize
+              "kafka.topic.size.bytes",
+              topicSize,
+              metricsTags
       );
       // Number of partitions
       double numPartition = (double) topicDescription.getPartitions().size();
       OrionServer.metricsGaugeNum(
-              "kafkaTopic",
-              "partition",
-              "num",
-              metricsTags,
-              numPartition
+              "kafka.topic.partition.count",
+              numPartition,
+              metricsTags
       );
       // Retention
       double retentionMs= Double.parseDouble(
               topicDescription.getTopicConfigs().getOrDefault("retention.ms", "0.0"));
       OrionServer.metricsGaugeNum(
-              "kafkaTopic",
-              "retention",
-              "millisecond",
-              metricsTags,
-              retentionMs
+              "kafka.topic.retention.ms",
+              retentionMs,
+              metricsTags
```

Comment:
I forgot to ask before. Would sending this metric help us with any alert?

Reply:
I remember we had a discussion that you want orion metrics helper not only send alerts, but also sends other information that can be helpful for debuging. Here are few examples.